tools/stonith_admin.c

```diff
@@ -448,14 +449,14 @@ handle_history(stonith_t *st, const char *target, int timeout, int quiet,
             continue;
         }
 
-        out->message(out, "stonith-event", hp, 1, history);
+        out->message(out, "stonith-event", hp, 1, stonith__later_succeeded(hp, history));
     }
 
     if (latest) {
         if (quiet && out->supports_quiet) {
             out->info(out, "%lld", (long long) latest->completed);
         } else if (!verbose) { // already printed if verbose
-            out->message(out, "stonith-event", latest, 0, NULL);
+            out->message(out, "stonith-event", hp, 1, FALSE);
```

Comment:
Using FALSE instead of NULL makes sense, but we still want to print latest, and not as part of full history (0)

Reply:
Yeah, I don't know why I changed latest to hp and 0 to 1 here.  Fixed.